### DISCUSSION TITLE
Getting current temperature, and whether or not it's currently raining.

### DISCUSSION THREAD

**Discussion by martincmartin** [weather-gov/api#661] [General]
Is there a way to find the current temperature, and whether or not it is currently raining, from a weather station?

My nearest station, https://api.weather.gov/stations/KBED , seems to have only a forecast.  Earlier I found some historical data, but the most recent point could be up to an hour old.  When the temperature changes by 8 degrees F per hour, data that's almost an hour old can be very misleading.

Similarly, is there a way to determine whether it is raining right now?  In the sense that it would be good to wear a raincoat or umbrella?

**Comment by scadergit** [comment]
You just need to add /observations/latest

**Reply by martincmartin** [reply]
Thanks!  This supports my point that the latest observation is up to an hour old.  The timestamp in /observations/latest is 2023-10-04T12:51:00+00:00, but the current UTC time is 13:27.

**Reply by akrherz** [reply]
That's generally how most of the federal ASOS reporting in METAR format works.  Once hourly reports are the only requirement.  They will generate SPECIal reports when conditions warrant, so that is why it isn't always just once per hour. Non-federal AWOS sites often are every 20 minutes, but won't do specials.  [MADIS HFMETAR](https://madis.ncep.noaa.gov/madis_OMO.shtml) exists, which has more frequent data for the federal sites, but it suffers from a number of issues, including latency, whole degree C reporting, and not "METAR" standard averaging periods that makes its data usage tricky.  The good news is that ASOS modernization is at hand, but how this all shakes out is not clear to me!
